packages/node_modules/samples/browser-single-party-call/app.js

```diff
@@ -227,3 +233,37 @@ document.getElementById('dialer').addEventListener('submit', (event) => {
       // Implement error handling here
     });
 });
+
+window.addEventListener('load', () => {
```

Comment:
Let's add a bunch of comments in here since this is a sample app. The more verbose the better since consumers will most likely be copy/pasting it into their apps.

Reply:
done!